packages/datagateway-search/src/search/myDataCheckBox.component.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Checkbox, FormControlLabel } from '@mui/material';
+import { ArrowTooltip } from 'datagateway-common';
+
+interface MyDataCheckBoxProps {
+  checked: boolean;
+  onChange: (checked: boolean) => void;
+}
+
+const MyDataCheckBox = ({
```

Comment:
This will need a test file (can be done through https://github.com/ral-facilities/datagateway/issues/1440 if needed)